add winston ai (#24)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

\subsubsection{Image generating models}

> \textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly Linguistic analysis analyses every text of its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}

Reviewer: ```suggestion
\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly linguistic analysis analyses every text for its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}
```

Reviewer: Is 'Linguistic analyse' a specific name ?
If not I would write it in lowercase

Reviewer 2: 'analyse' is not a noun at all, so in my opinion analysis is the right one here.

> Is 'Linguistic analysis' a specific name ?

Not really. The original source (see the bib-file) does once writes it uppercase, otherwise lowercase (in the middle of the sentence). I agree to @RaphaW and would orientate ourself on Wikipedia (https://en.wikipedia.org/wiki/Linguistics) and write it lowercase.

> \textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly linguistic analysis analyses every text for its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}

Reviewer 3: ```suggestion
\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness. Secondly is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}
```

> \textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. The two primary methods for checking if a text is AI written is firstly Linguistic analysis. Linguistic analysis analyses every text of its characteristics like repetitiveness. These characteristics are different between AI and humans. The other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}

Reviewer 2: ```suggestion
\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly Linguistic analysis analyses every text of its characteristics like repetitiveness. These characteristics are different between AI and humans. Secondly the other is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which than can be detected.\cite{WinstonAI}
```
